lib/css/CssParser.js

```diff
@@ -658,6 +713,11 @@ class CssParser extends Parser {
 					modeData = isLocalMode() ? "local" : "global";
 					isNextRulePrelude = true;
 					return end;
+				} else if (name === "@value") {
+					const pos = parseModulesValues(input, end);
+					const dep = new ConstDependency("", [start, pos]);
```

Comment:
We need to implement other dep, because `@value` can have url - `@value blue, red from "./colors.css`

Reply:
I want to implement @value importment using `:import`, like css loader

Reply:
Maybe I should implement `:import` firstly

Reply:
I think they will use different dependecies, because `@value` -> `:import {}` -> dependecy is not good for perf

Reply:
We can separate `@value` into `value definition` and `value imports`, similar to `postModulesValues`. This PR currently only implements `value definition`, which can be handled using `ConstDependency`. `value imports` can be managed through a dependency, perhaps called `CssIcssImportDependency`, which deals with the `:import` functionality.